admin-ui/src/pages/request-detail-page.tsx

```
@@ -47,6 +47,18 @@ import {
 
 const EMPTY = "—"
 
+function getQuotaLabel(item: AdminRequestItem): string {
+  if (item.premium_unlimited_after) {
+    return "∞"
+  }
+
+  if (item.premium_remaining_after != null) {
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)